controllers/hvpa_controller.go

```diff
@@ -1295,9 +1441,15 @@ func getThreshold(thresholdVals *autoscalingv1alpha1.ChangeParams, resourceType
 func (r *HvpaReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	ctx := context.Background()
 
-	// Fetch the Hvpa instance
+	// Fetch the Hvpa instance using unstructured object so that it is fetched using clientReader, and not the cacheReader
+	hvpaObj := &unstructured.Unstructured{}
+	hvpaObj.SetGroupVersionKind(controllerKindHvpa)
+
+	err := r.Get(ctx, req.NamespacedName, hvpaObj)
 	instance := &autoscalingv1alpha1.Hvpa{}
-	err := r.Get(ctx, req.NamespacedName, instance)
+	if errInt := runtime.DefaultUnstructuredConverter.FromUnstructured(hvpaObj.Object, instance); errInt != nil {
```

Comment:
Is there no other way to bypass the cache?

Reply:
Doesn't look like it. :/